pkgs/applications/science/electronics/pulseview/default.nix

```diff
@@ -1,6 +1,6 @@
 { stdenv, fetchurl, pkgconfig, cmake, glib, boost, libsigrok
 , libsigrokdecode, libserialport, libzip, udev, libusb1, libftdi1, glibmm
-, pcre, librevisa, python3, qtbase, qtsvg
+, pcre, librevisa, python3, qtbase, qtsvg, wrapQtAppsHook
 }:
 
 stdenv.mkDerivation rec {
```

Comment:
The preferred way to do this is to import `mkDerivation` (from `qt5`) and use that directly.